helm/lib/cortex-core/values.yaml

```diff
@@ -0,0 +1,282 @@
+      # Each scheduler step can specify its own dependencies on extractors and
+      # synced data.
+      #
+      # The `name` should correspond to a known scheduler plugin.
+      # Note that all scheduler steps will be executed in parallel. However,
+      # the order will be maintained when the weights are applied to the
+      # input Nova weights.
+      plugins:
+        - name: ""
+          # Options for the scheduler step.
+          options: {}
+          # Dependencies needed by this scheduler step.
+          dependencies: {} # e.g. {extractors: [my_nova_extractor]}
+
+  descheduler:
+    nova:
+      # If migrations should be executed.
+      disableDryRun: false
+      # The availability of the Nova endpoint to look for.
+      availability: public
+      # Configuration of the descheduler that runs periodically to de-schedule
+      # VMs based on the configured plugins.
+      # Each plugin can specify its own dependencies on extractors and synced data.
+      #
+      # The `name` should correspond to a known descheduler plugin.
+      plugins:
+        - name: ""
+          # Options for the descheduler plugin.
+          options: {}
+          # Dependencies needed by this descheduler plugin.
+          dependencies: {} # e.g. {extractors: [my_nova_extractor]}
+
+# Values passed to the service through a kubernetes secret.
+# The keys of these values should correspond to the keys used above in the conf.
+secrets:
+  # MQTT broker connection credentials.
+  mqtt:
+    username: cortex
+    password: secret
+  # Database connection credentials.
+  db:
+    user: postgres
+    password: secret
+  keystone:
+    # OpenStack login credentials also used by the OpenStack CLI.
+    projectName: ""
+    userDomainName: ""
+    projectDomainName: ""
+    username: ""
+    password: ""
+    # Optional, only required if OpenStack is configured with SSO.
+    # Note: the certificate will be used for all other OpenStack services.
+    sso:
+      # Certificate "public key".
+      cert: ""
+      # Certificate private key.
+      certKey: ""
+      # Whether the certificate is self-signed.
+      # If true, the certificate is not verified.
+      selfSigned: false
+  sync:
+    prometheus:
+      # Prometheus hosts to consider when syncing the metrics.
+      hosts:
+        - name: ""
+          # Prometheus URL to scrape metrics from.
+          url: ""
+          # The metric types to scrape from this Prometheus instance.
+          provides: []
+          # Optional, only required if Prometheus is configured with SSO.
+          sso:
+            # Certificate "public key".
+            cert: ""
+            # Certificate private key.
+            certKey: ""
+            # Whether the certificate is self-signed.
+            # If true, the certificate is not verified.
+            selfSigned: false
+
+# Generic modifiers added on the initial creation of this helm chart.
```

Comment:
if they're no used anywhere, I'd vote to remove them

Reply:
Will remove empty but necessary values to indicate they need to be confed.

Reply:
discussed